applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc

```diff
@@ -11,7 +11,7 @@ The *Observe* view in the *Developer* perspective provides options to monitor yo
 
 == Prerequisites
 
-* You have xref:../web_console/web-console.adoc#web-console-overview[logged in to the web console] and have switched to the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have xref:../web_console/web-console.adoc#web-console-overview[logged in to the web console] and have switched to the xref:../../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer* perspective].
```

Comment:
ERROR (applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc): "xref:../../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer* perspective]" appears to try to reference a file not included in the "openshift-enterprise" distro

Currently, this is the link throwing the build off. I think the single /../ was correct since this content wasn't moved into a new parent folder. I would try removing the /../ first. If that doesn't work, try removing the _web-console-overview from the path.